string  component_ref
---
string Returncode_t
string speech_text
